src/plugins/intel_cpu/src/nodes/kernels/x64/gather_uni_kernel.cpp

```diff
@@ -573,7 +573,7 @@ void jitUniGatherKernel<x64::avx512_core>::calcSrcShiftLong(Vmm* vAuxPool, bool
 }
 
 template <x64::cpu_isa_t isa>
-void jitUniGatherKernel<isa>::calcSrcShiftLongBlock(Vmm* vAuxPool, bool shiftFirst) {
+void jitUniGatherKernel<isa>::calcSrcShiftLongBlock(Vmm* /*vAuxPool*/, bool /*shiftFirst*/) {
```

Comment:
Also commented out vars, should be removed?

Reply:
See above